test/widgets/compose_box_checks.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:checks/checks.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:zulip/model/autocomplete.dart';
+import 'package:zulip/widgets/compose_box.dart';
+
+extension ComposeContentControllerChecks on Subject<ComposeContentController> {
+  Subject<TextEditingValue> get value => has((c) => c.value, 'value');
```

Comment:
This underlying getter is on ChangeNotifier, so it can go in a more general extension in `flutter_checks.dart`.

Reply:
`ValueNotifier`, not `ChangeNotifier`, I think, but yes: good idea. 🙂